tests/attestation_plugin/enc/enc.c

```diff
@@ -102,13 +111,29 @@ static void _test_sgx_remote()
         evidence_size,
         endorsements_size);
 
+    printf("====== verify evidence by passing NULL endorsements\n");
```

Comment:
```suggestion
    printf("====== verify evidence by passing NULL expected endorsements\n");
```

Reply:
This test case is passing `oe_verify_evidence()` with NULL endorsements. Expected endorsements are used to validate extracted endorsements related claims and they are not NULL here.

Reply:
Got it.